truenas/webui
# NAS-110981 / None / Fix text alignment on volume list

The problem is entity-form adds global styles when opening a wizard. It affects `.mat-content` class in the volume list so that styles should be isolated into components.

How to test?
- Open the Storage page, click the Import button, and then check the volume header it shouldn't be sticky to the top.
- Please check the video in JIRA ticket
